wgsl/index.bs

```diff
@@ -4261,8 +4264,9 @@ such that the redundant loads are eliminated.
     @group(0) @binding(1)
     var<storage,read_write> buf3: Buffer; // Can both read and write.
 
-    // Uniform buffer. Always read-only, and has more restrictive layout rules.
     struct ParamsTable {weight: f32}
+
+    // Uniform buffer. Always read-only, and has more restrictive layout rules.
```

Comment:
I wish github let you comment on lines outside the diff slight. I was referring too:

```
    @group(0) @binding(2)
    var<uniform> param: Params;    // A uniform buffer
```

in the example above.

Reply:
Ah, ok, I've duplicated it there.